terminusdb_client/woqlclient/woqlClient.py

```diff
@@ -805,13 +842,14 @@ def query(self, woql_query, commit_msg=None, file_dict=None):
             query_obj = self._generate_commit(commit_msg)
         else:
             query_obj = {}
-        if type(woql_query) != dict and hasattr(woql_query, "to_dict"):
-            woql_query = woql_query.to_dict()
-        woql_query["@context"] = self.conCapabilities.get_context_for_outbound_query(
+        if isinstance(woql_query, WOQLQuery):
```

Comment:
I think we have to keep the original way of checking this cause WOQLQuery is not allowed to be imported.